addons/website_hr_recruitment/static/src/js/website_hr_applicant_form.js

```diff
@@ -75,12 +76,12 @@ publicWidget.registry.hrRecruitment = publicWidget.Widget.extend({
     },
 
     async checkRedundant(target, field, messageContainerId, keepPreviousWarningMessage = false) {
-        const value = $(target).val();
+        const value = target.value;
         if (!value) {
             this.hideWarningMessage(target, messageContainerId);
             return;
         }
-        const job_id = $('#recruitment7').val();
+        const job_id = document.querySelector("#recruitment7").value;
```

Comment:
Could be renamed `jobId` - but it also matches the python name in the RPC... so, do as you wish.